src/index.d.ts

```diff
@@ -0,0 +1,67 @@
+interface IObject {
+  [key: string]: any;
+}
+
+interface IOptions {
+  messages?: any;
+  className?: any;
+  autoForceUpdate?: boolean;
+  element?: any;
+}
+
+type IRule = 'accepted' | 'after' | 'after_or_equal' | 'alpha' | 'alpha_space' | 'alpha_num' |' alpha_num_space' | 'alpha_num_dash' | 'alpha_num_dash_' | 'array' | 'before' | 'before_or_equal' | 'between' | 'boolean' | 'card_exp' | 'card_num' | 'currency' | 'date' | 'date_equals' | 'email' | 'in' | 'integer' | 'max' | 'min' | 'not_in' | 'not_regex' | 'numeric' | 'phone' | 'regex' | 'required' | 'size' | 'string' | 'typeof' | 'url';
```

Comment:
I think `alpha_num_dash_` is meant to be `alpha_num_dash_space`?

Reply:
yeah, that's right, my mistake...

Reply:
I noticed you have update the wrong spell word, so I am not going to open another pr.